src/Microsoft.DotNet.ProjectJsonMigration/Report/ProjectMigrationReport.cs

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections.Generic;
+using System.Linq;
+namespace Microsoft.DotNet.ProjectJsonMigration
+{
+    public class ProjectMigrationReport
```

Comment:
It'll probably be https://github.com/dotnet/roslyn-project-system/blob/master/src/Microsoft.VisualStudio.ProjectSystem.CSharp.VS/ProjectSystem/VS/Xproj/MigrateXprojProjectFactory.cs. At the very least, looking at that will provide the context for what needs to be updated, I may factor reading the log out.